src/polymer-cli.ts

```diff
@@ -122,7 +122,7 @@ export class PolymerCli {
     return Array.from(mergedArgs.values());
   }
 
-  run() {
+  run(): Promise<any> {
```

Comment:
Agreed. @FredKSchott and I talked about doing a big conversion of cli and polymer-build to async/await